bundles/org.openhab.binding.guntamatic/src/main/resources/OH-INF/i18n/guntamatic.properties

```diff
@@ -153,6 +189,46 @@ channel-type.guntamatic.controlProgram.state.option.0 = OFF
+channel-type.guntamatic.controlProgramWOManu.state.option.2 = WARMWATER
 channel-type.guntamatic.controlWwHeat.label = Trigger Warm Water Circle
 channel-type.guntamatic.controlWwHeat.description = Trigger Warm Water Circle of the Guntamatic Heating System (RECHARGE)
 channel-type.guntamatic.controlWwHeat.state.option.0 = RECHARGE
+
+# thing types
+
+thing-type.guntamatic.unknown.label = Guntamatic Unknown
+thing-type.guntamatic.unknown.description = Guntamatic Heating System. Use this, if your Heating System is none of the above. Please provide Feedback!
+thing-type.guntamatic.unknown.channel.controlExtraWwHeat0.label = Trigger Extra Warm Water Circle 0
+thing-type.guntamatic.unknown.channel.controlExtraWwHeat1.label = Trigger Extra Warm Water Circle 1
+thing-type.guntamatic.unknown.channel.controlExtraWwHeat2.label = Trigger Extra Warm Water Circle 2
+thing-type.guntamatic.unknown.channel.controlHeatCircProgram0.label = Set Heat Circle 0 Program
+thing-type.guntamatic.unknown.channel.controlHeatCircProgram1.label = Set Heat Circle 1 Program
+thing-type.guntamatic.unknown.channel.controlHeatCircProgram2.label = Set Heat Circle 2 Program
+thing-type.guntamatic.unknown.channel.controlHeatCircProgram3.label = Set Heat Circle 3 Program
+thing-type.guntamatic.unknown.channel.controlHeatCircProgram4.label = Set Heat Circle 4 Program
+thing-type.guntamatic.unknown.channel.controlHeatCircProgram5.label = Set Heat Circle 5 Program
+thing-type.guntamatic.unknown.channel.controlHeatCircProgram6.label = Set Heat Circle 6 Program
+thing-type.guntamatic.unknown.channel.controlHeatCircProgram7.label = Set Heat Circle 7 Program
+thing-type.guntamatic.unknown.channel.controlHeatCircProgram8.label = Set Heat Circle 8 Program
+thing-type.guntamatic.unknown.channel.controlProgram.label = Set Program
+thing-type.guntamatic.unknown.channel.controlWwHeat0.label = Trigger Warm Water Circle 0
+thing-type.guntamatic.unknown.channel.controlWwHeat1.label = Trigger Warm Water Circle 1
+thing-type.guntamatic.unknown.channel.controlWwHeat2.label = Trigger Warm Water Circle 2
+
+# channel types
+
+channel-type.guntamatic.controlProgramLog.label = Set Program
+channel-type.guntamatic.controlProgramLog.description = Set Program of the Guntamatic Heating System (OFF, NORMAL, WARMWATER)
+channel-type.guntamatic.controlProgramLog.state.option.0 = OFF
+channel-type.guntamatic.controlProgramLog.state.option.1 = NORMAL
+channel-type.guntamatic.controlProgramLog.state.option.2 = WARMWATER
+channel-type.guntamatic.controlProgramPellets.label = Set Program
+channel-type.guntamatic.controlProgramPellets.description = Set Program of the Guntamatic Heating System (OFF, NORMAL, WARMWATER, MANUAL)
+channel-type.guntamatic.controlProgramPellets.state.option.0 = OFF
+channel-type.guntamatic.controlProgramPellets.state.option.1 = NORMAL
+channel-type.guntamatic.controlProgramPellets.state.option.2 = WARMWATER
+channel-type.guntamatic.controlProgramPellets.state.option.8 = MANUAL
```

Comment:
Is controlProgramPellets a dynamic channel ?

Reply:
no - this is outdated imfo. see comment above